diff --git a/optuna/storages/_rdb/alembic/versions/v3.0.0.c.py b/optuna/storages/_rdb/alembic/versions/v3.0.0.c.py
index f96df8609..c506272ae 100644
--- a/optuna/storages/_rdb/alembic/versions/v3.0.0.c.py
+++ b/optuna/storages/_rdb/alembic/versions/v3.0.0.c.py
@@ -87,6 +87,7 @@ def upgrade():
             server_default=None,
         )
 
+    raise Exception("This migration is not complete. Please fix the inconsistent state.")
     session = orm.Session(bind=bind)
     try:
         records = session.query(IntermediateValueModel).all()
